client/sprotty-ecore/css/diagram.css

```diff
@@ -146,23 +165,19 @@ svg {
 }
 
 .ecore-edge>.triangle.inheritance {
-    fill: white;
-}
-
-.ecore-edge.inheritance {
```

Comment:
why is this removed completely?

Reply:
I talked to @JonasHelming and he thought, that the inheritance edges should have the same color as the other edges. And since the only think the ecore-edge.inheritance attribute did was changing the color i removed it.